Tasks/RunRemoteShellCommandsSSH/task.json

```diff
@@ -0,0 +1,89 @@
+{
+    "id": "91443475-df55-4874-944b-39253b558790",
+    "name": "RunRemoteShellCommandsSSH",
+    "friendlyName": "Remote Shell Commands",
+    "description": "Run shell commands on a remote machine using SSH.",
+    "helpMarkDown": "",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "demands": [
+     ],
+    "instanceNameFormat": "Run shell commands on remote machine using SSH",
+    "groups": [
+        {
+            "name": "connection",
+            "displayName": "Connection",
+            "isExpanded": true
+        }
+    ],
+    "inputs": [
+        {
+            "name": "hostname",
+            "type": "string",
+            "label": "Hostname",
+            "defaultValue": "",
+            "required": true,
+            "groupName": "connection",
+            "helpMarkDown": "Hostname or IP of the remote machine."
+        },
```

Comment:
Nope.  We should go the other way and make it a service endpoint so it's manageable.  It's much more critical that if a password changes, I can go to one place.  If they invoke cmds on 2 machines, you want to see success/fail and output for each.  Many machines == many instances of the task with clear communication and easy management of credentials (not baked into the task)